pd/src/genesis.rs

```diff
@@ -71,79 +63,59 @@ impl FromStr for GenesisAddr {
     }
 }
 
-mod helpers {
+pub mod helpers {
```

Comment:
Rather than making this module public, I think it could be eliminated / have its contents folded into the main `genesis` module.  Originally, we were using serde on remote types, so it made sense to have a helpers module to contain the serde code, but now we can just expose the types directly.